app/javascript/components/student/tracks-list/Track.jsx

```diff
@@ -29,7 +29,8 @@ export function Track({ track }) {
           </ul>
           <ul className="--tags">
             {track.tags.slice(0, 3).map((tag) => {
-              return <li key={tag}>{tag.split(':')[1]}</li>
+              // TODO: find a nice solution to map tags to their human-readable form
+              return <li key={tag}>{tag.split('/')[1]}</li>
```

Comment:
This is now an open issue. I _could_ use the TAG_OPTIONS that are passed with the data, but I'm not entirely sure that is the best way to go.